encoding/thrift/thriftrw-plugin-yarpc/exception.go

```diff
@@ -61,26 +60,46 @@ package <$pkgname>
 `
 
 var (
-	_errorCodeToTypeName = map[yarpcerrors.Code]string{
-		yarpcerrors.CodeCancelled:          "yarpcerrors.CodeCancelled",
-		yarpcerrors.CodeUnknown:            "yarpcerrors.CodeUnknown",
-		yarpcerrors.CodeInvalidArgument:    "yarpcerrors.CodeInvalidArgument",
-		yarpcerrors.CodeDeadlineExceeded:   "yarpcerrors.CodeDeadlineExceeded",
-		yarpcerrors.CodeNotFound:           "yarpcerrors.CodeNotFound",
-		yarpcerrors.CodeAlreadyExists:      "yarpcerrors.CodeAlreadyExists",
-		yarpcerrors.CodePermissionDenied:   "yarpcerrors.CodePermissionDenied",
-		yarpcerrors.CodeResourceExhausted:  "yarpcerrors.CodeResourceExhausted",
-		yarpcerrors.CodeFailedPrecondition: "yarpcerrors.CodeFailedPrecondition",
-		yarpcerrors.CodeAborted:            "yarpcerrors.CodeAborted",
-		yarpcerrors.CodeOutOfRange:         "yarpcerrors.CodeOutOfRange",
-		yarpcerrors.CodeUnimplemented:      "yarpcerrors.CodeUnimplemented",
-		yarpcerrors.CodeInternal:           "yarpcerrors.CodeInternal",
-		yarpcerrors.CodeUnavailable:        "yarpcerrors.CodeUnavailable",
-		yarpcerrors.CodeDataLoss:           "yarpcerrors.CodeDataLoss",
-		yarpcerrors.CodeUnauthenticated:    "yarpcerrors.CodeUnauthenticated",
+	_errorCodeToTypeName = map[string]string{
```

Comment:
Should we still call this `errorCodeToTypeName`? It's not really mapping a code to the type name anymore.

Reply:
yup good call, I'll rename that